emoji/core.py

```diff
@@ -45,3 +46,31 @@ def replace(match):
             return unicode_codes.EMOJI_UNICODE.get(match.group(1), match.group(1))
 
     return pattern.sub(replace, string)
+
+def demojize(string):
+
+    """Replace unicode emoji in a string with emoji shortcodes. Useful for storage.
+
+    :param string: String contains unicode characters. MUST BE UNICODE.
+
+        >>> import emoji
+        >>> print(emoji.emojize("Python is fun :thumbs_up_sign:"))
+        Python is fun 👍
+        >>> print(emoji.demojize(u"Python is fun 👍"))
+        Python is fun :thumbs_up_sign:
+        >>> print(emoji.demojize("Unicode is tricky 😯".decode('utf-8')))
+        Unicode is tricky :hushed_face:
+    """
+    # Build emoji regexp once
```

Comment:
Perhaps it would be nice to have this as a separate initialization function? That way the regexp could be used elsewhere. (I'm thinking of submitting a PR with a function to strip emojis).